sonic_data_client/db_client.go

```diff
@@ -62,7 +62,7 @@ type Stream interface {
 var UseRedisLocalTcpPort bool = false
 
 // redis client connected to each DB
-var Target2RedisDb = make(map[string]*redis.Client)
+var Target2RedisDb = make(map[string]map[string]*redis.Client)
```

Comment:
I am new to go, but could you explain why this map[string] is repeated here?

Reply:
@roweisch yes. Basically we have {dbname} to {redis client}  mapping .. now we have {namespace, dbname} to {redis client} mapping.